spatialconnect/build.gradle

```diff
@@ -70,3 +70,12 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
 }
+
+license {
+    ext.year = Calendar.getInstance().get(Calendar.YEAR)
+    ext.name = 'Boundless http://boundlessgeo.com'
+    header rootProject.file('NOTICE')
```

Comment:
Oh I see you are building the correct license notice as part of packaging step ... tricky